### PR TITLE
OSDOCS-5136: add rhel9 support information

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.13

Issue: https://issues.redhat.com/browse/OSDOCS-5136

Link to docs preview:

- features and enhancements: http://file.rdu.redhat.com/jdohmann/OSDOCS-5136/release_notes/ocp-4-13-release-notes.html#ocp-4-13-rhel-9-considerations
- deprecated features: http://file.rdu.redhat.com/jdohmann/OSDOCS-5136/release_notes/ocp-4-13-release-notes.html#ocp-4-13-rhel-9-device-driver-deprecation

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@mike-nguyen could you PTAL for QE ack? This PR adds the RHEL 9 FAQ information.

cc: @control-d, @marrusl

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -52,6 +52,17 @@ This release adds improvements related to the following components and concepts.
...
 {op-system} now uses {op-system-base-full} 9.2 packages in {product-title} {product-version}. This enables you to have the latest fixes, features, and enhancements, as well as the latest hardware support and driver updates.
 
+[id="ocp-4-13-rhel-9-considerations"]
+===== Considerations for upgrading to {product-title} with {op-system-base} 9.2
+
+With this release, {product-title} {product-version} introduces a {op-system-base} 9.2 based {op-system} and there are some considerations you must make before upgrading:
+
+* Some component configuration options and services might have changed between {op-system-base} 8.6 and {op-system-base} 9.2, which means existing machine configuration files might no longer be valid.
+
+* {op-system-base} 6 base image containers are not supported on {op-system} container hosts, but are supported on {op-system-base} 8 worker nodes. For more information, see the link:https://access.redhat.com/support/policy/rhel-container-compatibility[Red Hat Container Compatibility] guide.
```
Comment (GroceryBoyJr) [inline release_notes/ocp-4-13-release-notes.adoc:62]:
```suggestion
* {op-system-base} 6 base image containers are not supported on {op-system} container hosts, but are supported on {op-system-base} 8 worker nodes. For more information, see the link:https://access.redhat.com/support/policy/rhel-container-compatibility[Red Hat Container Compatibility] matrix.
```

**Review by GroceryBoyJr — COMMENTED** [review]
@jldohmann only one tiny nit, your call. Otherwise LGTM!

**Review by JoeAldinger — COMMENTED** [review]
/lgtm

**Comment by JoeAldinger** [comment]
CC @sdodson

**Comment by jldohmann** [comment]
847616996e..1ccce00c26 implement peer review feedback

**Pull request merged**